### DISCUSSION TITLE
Is there any way / plan to swap dateTime (adapter) form date-fns to Luxon ?

### DISCUSSION THREAD

**Discussion by apudiu** [gpbl/react-day-picker#1953] [Ideas]
I really hate to work with date-fns as it works with the **horrible** JS **Date**. Is there any way to switch to luxon? or any plan to do so?

**Comment by gpbl** [comment]
Ahaha @apudiu don't complain with us about the `Date` object, I've been fighting it since I started this project :) 

We do wish to add support with multiple date library (see #1087) but no progress at the moment. I think most of us find convenient to work with date-fns and native `Date` objects.
